app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -619,21 +620,33 @@ private function extractTraits(array $nft): array
         /** @var array<array{trait_type: string | null, value: string | null, display_type: string | null}> $props */
         $props = Arr::get($nft, 'metadata.attributes', Arr::get($nft, 'metadata.properties', []));
 
-        return collect($props)
-            ->filter(function ($item) {
-                return ! empty(Arr::get($item, 'trait_type')) && ! empty(Arr::get($item, 'value'));
-            })
-            ->map(function ($item) {
-                $value = strval($item['value']);
-                $displayType = TraitDisplayType::fromAlchemyDisplayType(Arr::get($item, 'display_type'), $value);
-
-                return [
-                    'name' => $item['trait_type'],
-                    'value' => $value,
-                    'displayType' => $displayType,
-                ];
-            })
-            ->toArray();
+        try {
+            return collect($props)
+                ->filter(function ($item) {
+                    return ! empty(Arr::get($item, 'trait_type')) && ! empty(Arr::get($item, 'value')) && ! is_array(Arr::get($item, 'value'));
+                })
+                ->map(function ($item) {
+                    $value = strval($item['value']);
+                    $displayType = TraitDisplayType::fromAlchemyDisplayType(Arr::get($item, 'display_type'), $value);
+
+                    return [
+                        'name' => $item['trait_type'],
+                        'value' => $value,
+                        'displayType' => $displayType,
+                    ];
+                })
+                ->toArray();
+        } catch (Throwable $e) {
```

Comment:
technically this shouldn't really happen anymore, will see if I can manage to write a test to get into this catch

Reply:
negative, removing it